snapcraft/tests/commands/test_update.py

```diff
@@ -106,3 +132,28 @@ def test_update_with_no_content_length_is_supported(self):
         self.assertThat(result.exit_code, Equals(0))
         self.assertThat(self.parts_yaml, FileExists())
         self.assertThat(self.headers_yaml, FileExists())
+
+    @mock.patch('snapcraft.internal.lxd.Containerbuild._container_run')
+    @mock.patch('os.getuid')
+    def test_update_containerized_exists_running(self,
+                                                 mock_getuid,
+                                                 mock_container_run):
+        mock_container_run.side_effect = lambda cmd, **kwargs: cmd
+        mock_getuid.return_value = 1234
+        fake_lxd = fixture_setup.FakeLXD()
+        self.useFixture(fake_lxd)
+        # Container was created before and is running
+        fake_lxd.name = 'local:snapcraft-snap-test'
+        fake_lxd.status = 'Running'
+        fake_logger = fixtures.FakeLogger(level=logging.INFO)
+        self.useFixture(fake_logger)
```

Comment:
I don't think you need this fake logger setup here; or what is it intended for as there are no checks for it. `self.run_command` would return an object with an `output` and `exit_code` attribute.

Reply:
You're right! Thanks for pointing it out! I'd copied it because it's used in other tests, but actually the output is the same - we could probably simplify a few other tests this way at some point.
I dropped the logger and added a check for exit code (not sure if it's really necessary here, but can't hurt to have it).